File: Latihan3-py/RAM.py
class RAM:
    # Constructor
    def __init__(self):
        self.__capacity = 0
        self.__price = 0
    
    # Get-set Capacity
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    # Get-set Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

File: Latihan3-cpp/RAM.cpp
class RAM{

    private:

    int capacity;
    int price;

    public:

    // Constructor
    RAM(){
    }

    RAM(int capacity, int price){
        this->capacity = capacity;
        this->price = price;
    }

    // Get-set Capacity
    void setCapacity(int capacity){
        this->capacity = capacity;
    }

    int getCapacity(){
        return this->capacity;
    }

    // Get-set Price
    void setPrice(int price){
        this->price = price;
    }

    int getPrice(){
        return this->price;
    }

    // Deconstructor
    ~RAM(){
    }

};